### PR TITLE
feature: new endpoint for organization admins to get pending invitations

### PR DESCRIPTION
**Pull request by lucasmenendez**
check the updated [api docs](https://github.com/vocdoni/saas-backend/blob/f/pending_org_invititations/api/docs.md#-list-pending-invitations).

### CODE REVIEW

File: api/docs.md [hunk 1]
```diff
@@ -598,6 +601,35 @@ Only the following parameters can be changed. Every parameter is optional.
 | `409` | `40901` | `duplicate conflict` |
 | `500` | `50002` | `internal server error` |
 
+### ⏳ List pending invitations
+
+* **Path** `/organizations/{address}/members/pending`
+* **Method** `POST`
```
Comment (elboletaire) [inline api/docs.md:607]: Isn't it GET?

**Review by elboletaire — CHANGES_REQUESTED** [review]

File: api/docs.md (hunk 1 above)
Comment (lucasmenendez) [reply api/docs.md:607]: yes, it is

**Review by elboletaire — APPROVED** [review]

File: api/routes.go [hunk 2]
```diff
@@ -50,6 +50,8 @@ const (
 	organizationAddMemberEndpoint = "/organizations/{address}/members"
 	// POST /organizations/{address}/members/invite/accept to accept the invitation
 	organizationAcceptMemberEndpoint = "/organizations/{address}/members/accept"
+	// GET /organizations/{address}/members/pending to get the pending members
+	organizationPendingMemberEndpoint = "/organizations/{address}/members/pending"
```
Comment (emmdim) [inline api/routes.go:54]: @lucasmenendez  organizationPendingMember{s}Endpoint i gues :-P

File: api/organizations.go [hunk 3]
```diff
@@ -397,6 +397,45 @@ func (a *API) acceptOrganizationMemberInvitationHandler(w http.ResponseWriter, r
...
+	if !ok {
+		ErrNoOrganizationProvided.Write(w)
+		return
+	}
+	if !user.HasRoleFor(org.Address, db.AdminRole) {
+		ErrUnauthorized.Withf("user is not admin of organization").Write(w)
+		return
+	}
+	// check if the user is already verified
+	if !user.Verified {
```
Comment (emmdim) [inline api/organizations.go:418]: @lucasmenendez  why is that relevenat here?

File: api/routes.go (hunk 2 above)
Comment (lucasmenendez) [reply api/routes.go:54]: yep, i gues{s} :P

File: api/organizations.go (hunk 3 above)
Comment (lucasmenendez) [reply api/organizations.go:418]: to prevent that not verified users to perform actions

Comment (emmdim) [reply api/organizations.go:418]: But what is the action in this case? I mean you have check that the user is Admin, and your a getting the pending invitations...

Comment (lucasmenendez) [reply api/organizations.go:418]: The operation is to get a list of pending invitations, and yes, I've already checked that the user has an admin role for the requested org, but I think if the account is not yet verified, the user should not perform any operation, not even list invitations.

**Comment by emmdim** [comment]
LGTM

File: api/organizations.go (hunk 3 above)
Comment (emmdim) [reply api/organizations.go:418]: @lucasmenendez will create new PR that will improve the authorization/verifcation  helpers of the API

**Pull request merged**